callisto_core/delivery/forms.py

```diff
@@ -111,5 +111,9 @@ def _decrypt_report(self):
         return self.data["key"]
 
     def _decryption_failed(self):
+        # XXX (lojikil//Stefan Trail of Bits) the `report` property includes
+        # a token & UUID for reports; technically the UUID is in the URL, so
+        # that's ok, but the property would leak the token to logs; is that
+        # problematic?
```

Comment:
`self.message_key_error_log` seems to output everything - we can update it to just output `report.uuid`, I think.